sequencer/src/lib.rs

```diff
@@ -64,13 +68,14 @@ impl Node {
             &[&Tx::get_cf(), &compute::Result::get_cf(), &compute::ResultReference::get_cf()],
         )?;
         let (sender, receiver) = mpsc::channel(100);
-        let seq_server = SequencerServer::new(sender, config.whitelist.users.clone(), db);
+        let db = Arc::new(Mutex::new(db));
```

Comment:
We don't need `Mutex`

Reply:
I think we still need `Mutex`.

The `Db` in the `sequencer` is secondary db, for read/query purpose.
Current mechanism is: create one main thread for serving read/query request, and create one extra thread for refreshing in interval.
I believe that `Db` should be locked when it is in `refresh` mode since it is DB write.
Also, it is good to lock the `Db` when serving read/query request, so that it is not going to `refresh` thread.

What do you think? @lazovicff

Reply:
Done da968ea
(Thanks for local testing & confirmation! @lazovicff )